pelican/tests/test_settings.py

```diff
@@ -319,3 +320,11 @@ def test_coerce_overrides(self):
             'THEME_STATIC_DIR': 'theme',
         }
         self.assertDictEqual(overrides, expected)
+
+    def test_stringify_plugins(self):
```

Comment:
It would probably makes more sense to move this test into `test_plugins.py`. And it would be nice to extend with some namespace and normal plugin tests (there are some already there).